webapp-django/crashstats/crashstats/decorators.py

```diff
@@ -4,11 +4,28 @@
 from django.http import HttpResponseBadRequest, Http404
 from django.shortcuts import redirect
 from django.core.urlresolvers import reverse
+from django.shortcuts import render
 
 from . import utils
 from crashstats.base import ga
 
 
+def decorate_bad_request(view):
```

Comment:
What about "dressup" or "prettify" or "htmlify" or "redress"?
To me, "decorate" sounds like you're making something pretty. And it's very much about making a plain text thing into HTML, but "htmlify" just sounds like jargon :)